### PR TITLE
2.13.1 with old pinnings

### PR DESCRIPTION
**Pull request by xhochy**
Let's at least get this release out. We'll probably have to wait 1-2 more releases until we can loosen the protobuf pin.

Fixes #345
Fixes #329
(doesn't fix 335)

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -140,6 +148,8 @@ if [[ ${cuda_compiler_version} != "None" ]]; then
         export TF_CUDA_COMPUTE_CAPABILITIES=sm_35,sm_50,sm_60,sm_62,sm_70,sm_72,sm_75,sm_80,sm_86,compute_86
     elif [[ ${cuda_compiler_version} == 11.2 ]]; then
         export TF_CUDA_COMPUTE_CAPABILITIES=sm_35,sm_50,sm_60,sm_62,sm_70,sm_72,sm_75,sm_80,sm_86,compute_86
+    elif [[ ${cuda_compiler_version} == 11.8 ]]; then
+        export TF_CUDA_COMPUTE_CAPABILITIES=sm_35,sm_50,sm_60,sm_62,sm_70,sm_72,sm_75,sm_80,sm_86,compute_86
```
Comment (xhochy) [inline recipe/build.sh:152]: @jakirkham @hmaarrfk @njzjz Does anyone of you know what the correct values would be here?

**Comment by xhochy** [comment]
This builds locally for me on Linux in the CPU-only and the CUDA 11.8 version. If someone can tell me what the correct configuration for CUDA 11.8 is, I can build and upload the full matrix.

File: recipe/build.sh (hunk 1 above)
Comment (hmaarrfk) [reply recipe/build.sh:152]: maybe follow @carterbox 's work https://github.com/conda-forge/magma-feedstock/blob/a9a5595de6a48f35515e731533d0a10818139a1a/recipe/build.sh#L22

Comment (njzjz) [reply recipe/build.sh:152]: CMake lists all architectures:
https://gitlab.kitware.com/cmake/cmake/-/blob/4d90f65b246fe7f9f9fc1d34fb985cd67b992aa9/Modules/CUDA/architectures.cmake#L50

```suggestion
        export TF_CUDA_COMPUTE_CAPABILITIES=sm_35,sm_50,sm_60,sm_62,sm_70,sm_72,sm_75,sm_80,sm_86,sm_87,sm_89,sm_90,compute_90
```

**Comment by xhochy** [comment]
Packages are ready. Linux packages are on `uwe.korn-tf-gpu` and macOS Packages on `uwe.korn-tf-experimental`. Logs:

- [linux_64_c_compiler_version11cuda_compilernvcccuda_compiler_version11.8cxx_compiler_version11numpy1.22python3.8.____cpython-log.txt](https://gist.github.com/xhochy/5d4c9b290f78c902919f6d811a8cfb0f)
- [linux_64_c_compiler_version11cuda_compilernvcccuda_compiler_version11.8cxx_compiler_version11numpy1.22python3.9.____cpython-log.txt](https://gist.github.com/xhochy/ce45ae39e9dbb372f41da9840979d586)
- [linux_64_c_compiler_version11cuda_compilernvcccuda_compiler_version11.8cxx_compiler_version11numpy1.22python3.10.____cpython-log.txt](https://gist.github.com/xhochy/1c8db28905d358227ee3bbcabe6b728d)
- [linux_64_c_compiler_version12cuda_compilerNonecuda_compiler_versionNonecxx_compiler_version12numpy1.23python3.11.____cpython-log.txt](https://gist.github.com/xhochy/0e0cf2adcf886dbc958228007e4449f6)
- [linux_64_c_compiler_version11cuda_compilernvcccuda_compiler_version11.8cxx_compiler_version11numpy1.23python3.11.____cpython-log.txt](https://gist.github.com/xhochy/3d300b7ba297677cb80dd71e413ab928)
- [linux_64_c_compiler_version12cuda_compilerNonecuda_compiler_versionNonecxx_compiler_version12numpy1.22python3.10.____cpython-log.txt](https://gist.github.com/xhochy/4160322ba6b6b98671e23c460178b85b)
- [linux_64_c_compiler_version12cuda_compilerNonecuda_compiler_versionNonecxx_compiler_version12numpy1.22python3.9.____cpython-log.txt](https://gist.github.com/xhochy/300c6dcb4c78454eddf2800d7e25c9f1)
- [linux_64_c_compiler_version12cuda_compilerNonecuda_compiler_versionNonecxx_compiler_version12numpy1.22python3.8.____cpython-log.txt](https://gist.github.com/xhochy/ccf12974631a7981687ba2e3cde74094)
- [linux_64_c_compiler_version10cuda_compilernvcccuda_compiler_version11.2cxx_compiler_version10numpy1.22python3.9.____cpython-log.txt](https://gist.github.com/xhochy/431351ce85bff6107dbe4aa11eb90498)
- [linux_64_c_compiler_version10cuda_compilernvcccuda_compiler_version11.2cxx_compiler_version10numpy1.22python3.10.____cpython-log.txt](https://gist.github.com/xhochy/871fe0f1b94fdf55d6f11945fccbbcff)
- [linux_64_c_compiler_version10cuda_compilernvcccuda_compiler_version11.2cxx_compiler_version10numpy1.22python3.8.____cpython-log.txt](https://gist.github.com/xhochy/20f32e5f3ee1a0d91a079959d8415b28)
- [linux_64_c_compiler_version10cuda_compilernvcccuda_compiler_version11.2cxx_compiler_version10numpy1.23python3.11.____cpython-log.txt](https://gist.github.com/xhochy/2e37b7a510554e4225f401ea3872eeb2)
- [osx_arm64_numpy1.23python3.11.____cpython.log](https://gist.github.com/xhochy/e647fb9b48ce10272d0faf7e32aebe5a)
- [osx_arm64_numpy1.22python3.10.____cpython.log](https://gist.github.com/xhochy/7757956f972ee655f0c2305324e0fe2b)
- [osx_arm64_numpy1.22python3.9.____cpython.log](https://gist.github.com/xhochy/606d44d5663c93bcea9264f660168ce1)
- [osx_arm64_numpy1.22python3.8.____cpython.log](https://gist.github.com/xhochy/254c8cafa8a3026f3142bb6fcd8af1a3)
- [osx_64_numpy1.23python3.11.____cpython.log](https://gist.github.com/xhochy/28d59082292bec93d50023447a0ef6a5)
- [osx_64_numpy1.22python3.8.____cpython.log](https://gist.github.com/xhochy/4ec06dbe2b918ada0096b82fd742e3dd)
- [osx_64_numpy1.22python3.9.____cpython.log](https://gist.github.com/xhochy/5c3884c17ef4df8b1256ee7a1c930f35)
- [osx_64_numpy1.22python3.10.____cpython.log](https://gist.github.com/xhochy/a7fa88e27784e65a579031d5667507fc)

**Comment by xhochy** [comment]
@h-vetinari @hmaarrfk Can you copy and merge?

**Comment by h-vetinari** [comment]
> @h-vetinari @hmaarrfk Can you copy and merge?

Never copied artefacts yet, so I'll defer to @hmaarrfk.

**Comment by xhochy** [comment]
The copy command is listed here: https://github.com/conda-forge/cfep/blob/main/cfep-03.md#specification I would also do this but I don't know whether that is OK according to the CFEP.

**Comment by hmaarrfk** [comment]
you can also upload directly.

my instructions are in a comment https://github.com/conda-forge/tensorflow-feedstock/issues/291

many thanks

**Comment by hmaarrfk** [comment]
@xhochy I feel like a while back we discussed that given the amount of core devs that were watching this package, it would be OK for them to directly upload.

Its pretty hard to store this many packages (even locally) so sometimes it has been prohibitive for me to keep creating random anaconda accounts.

**Comment by jonas-eschle** [comment]
Amazing, thanks a lot for the work!

**Pull request merged**